serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/ServiceRecordFactory.java

```diff
@@ -88,26 +90,22 @@ public ServiceRecordFactory(ServiceregistryProperties properties,
         return Optional.empty();
     }
 
-    @SuppressWarnings("squid:S1166") // Suppress Sonar due to no rethrow/log from certificate ex.
+    public Optional<ServiceRecord> createFiksErrorRecord(String orgnr) {
+        return Optional.of(ErrorServiceRecord.create(DPF));
+    }
+
+    @HystrixCommand(fallbackMethod = "createEduErrorRecord")
     public Optional<ServiceRecord> createEduServiceRecord(String orgnr) {
         Endpoint ep;
         try {
             ep = elmaLookupService.lookup(NORWAY_PREFIX + orgnr);
         } catch (EndpointUrlNotFound endpointUrlNotFound) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)